WcaOnRails/spec/spec_helper.rb

```diff
@@ -84,4 +84,15 @@
   # test failures related to randomization by passing the same `--seed` value
   # as the one that triggered the failure.
   Kernel.srand config.seed
+
+  config.before(:suite) do
+    %w(Countries Continents Events Rounds Formats teams).each do |table|
+      # -- Uncomment to regenerate --
+      # ActiveRecord::Base.connection.execute("TRUNCATE #{table};")
+
+      if ActiveRecord::Base.connection.execute("SELECT COUNT(*) FROM #{table}").first == [0]
+        load "#{Rails.root}/db/seeds/#{table.underscore}.seeds.rb"
```

Comment:
Yes I think it's per test suite. Here's the existing code:
```
  config.before(:each) do
    DatabaseCleaner.start
    FactoryGirl.create(:team, friendly_id: 'software', name: 'Software Team', description: 'Does software')
    FactoryGirl.create(:team, friendly_id: 'results', name: 'Results Team', description: 'Posts results')
    FactoryGirl.create(:team, friendly_id: 'wrc', name: 'WRC Team', description: 'Regulations')
    FactoryGirl.create(:team, friendly_id: 'wdc', name: 'WDC Team', description: 'Disciplinary Committee')
  end
```

From memory the whole test suite takes around 90 to 120 seconds to complete. I haven't measured the time the data loads take. I'll check when I get a chance.